pkgs/top-level/all-packages.nix

```diff
@@ -22615,8 +22622,6 @@ with pkgs;
 
   libqalculate = callPackage ../development/libraries/libqalculate { };
 
-  libqt5pas = libsForQt5.callPackage ../development/compilers/fpc/libqt5pas.nix { };
```

Comment:
An alias for `libqt5pas` should be added to `pkgs/top-level/aliases.nix` for backwards compatibility